Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:14.10
+
+WORKDIR /vpn
+
+RUN apt-get install -y curl
+RUN curl -sL http://nginx.org/keys/nginx_signing.key | apt-key add -
```

Comment:
Note that this provides no security if the key signatures or fingerprint is not verified since the key can be replaced by a MiTM